src/server/procedures/myForest/stats.ts

```diff
@@ -30,6 +32,32 @@ export const stats = procedure
       });
     }
 
+    const groupTreecounterData = await prisma.$queryRaw<
```

Comment:
It could be more readable to modify the query like this:
```
SELECT p.guid as profile_id
FROM profile p
         INNER JOIN treecounter t ON p.treecounter_id = t.id
         INNER JOIN treecounter_group child ON child.treecounter_id = t.id
         INNER JOIN treecounter_group parent ON child.root_id = parent.id
WHERE parent.slug = 'gvtrucknet-grosse-vehne';
```

Reply:
I agree, I'll make that change